### ISSUE TITLE
Cannot create children for a parent that is in a different thread

### ISSUE THREAD

**Issue by Connoropolous** [holochain/holosqape#39]
<img width="1561" alt="screen shot 2018-11-04 at 12 41 09 pm" src="https://user-images.githubusercontent.com/1409121/47967752-f77bc000-e02e-11e8-8bd2-fbb261e2788d.png">

**Comment by guillemcordoba** [comment]
Same issue for me, is there some workaround/dependency missing? 
Is Qt development installation necessary?

**Comment by Connoropolous** [comment]
@lucksus any ideas on this one?

**Comment by lucksus** [comment]
Yes, I'm pretty sure it's my fix for not working websockets in hcshell. I've made the interactive shell live in its own thread and introduced this bug. Wanted to fix this yesterday but didn't get to it. This is the next top most item on my priority list. Not sure if I will have time over the weekend but pretty sure it's fixed on Monday evening. Guess we should also build another release after this fix (?).
@Connoropolous

**Comment by lucksus** [comment]
I am now looking into this. I do see the same error message but everything does work anyway. Does this actually cause a problem? @Connoropolous @guillemcordoba

**Comment by Connoropolous** [comment]
Things do appear to work anyways... it just appears that something went
wrong, and it certainly seems concerning
